docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md

```diff
@@ -0,0 +1,208 @@
+| memoryLimit | The size of the batch, in bytes, before we flush the batch                                                                                                     | `10485760` (10 MB) |
+
+With the default configuration, the exporter will aggregate records and flush them to Elasticsearch:
+
+1. When it has aggregated 1000 records.
+2. When the batch memory size exceeds 10 MB.
+3. Five seconds have elapsed since the last flush (regardless of how many records were aggregated).
+
+### Retention
+
+A retention policy can be set up to delete old data.
+When enabled, this creates an Index Lifecycle Management (ILM) Policy that deletes the data after the specified `minimumAge`.
+All index templates created by this exporter apply the created ILM Policy.
+
+| Option     | Description                                                                  | Default                         |
+| ---------- | ---------------------------------------------------------------------------- | ------------------------------- |
+| enabled    | If `true` the ILM Policy is created and applied to the index templates       | `false`                         |
+| minimumAge | Specifies how old the data must be, before the data is deleted as a duration | `30d`                           |
+| policyName | The name of the created and applied ILM policy                               | `zeebe-record-retention-policy` |
+
+:::note
+The duration can be specified in days `d`, hours `h`, minutes `m`, seconds `s`, milliseconds `ms`, and/or nanoseconds `nanos`.
+:::
+
+### Authentication
+
+Providing these authentication options will enable Basic Authentication on the exporter.
+
+| Option   | Description                   | Default |
+| -------- | ----------------------------- | ------- |
+| username | Username used to authenticate | N/A     |
+| password | Password used to authenticate | N/A     |
+
+## Example
+
+Here is an example configuration of the exporter:
+
+```yaml
+---
+exporters:
+  elasticsearch:
+    # Elasticsearch Exporter ----------
+    # An example configuration for the elasticsearch exporter:
+    #
+    # These setting can also be overridden using the environment variables "ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_..."
+    #
+
+    className: io.camunda.zeebe.exporter.ElasticsearchExporter
+
+    args:
+      # A comma separated list of URLs pointing to the Elasticsearch instances you wish to export to.
+      # For example, if you want to connect to multiple nodes for redundancy:
+      # url: http://localhost:9200,http://localhost:9201
+      url: http://localhost:9200
+
+      bulk:
+        delay: 5
+        size: 1000
+        memoryLimit: 10485760
+
+      authentication:
+        username: elastic
+        password: changeme
+
+      index:
+        prefix: zeebe-record
+        createTemplate: true
```

Comment:
🔧 @remcowesterhoud  Please also add the retention example

Reply:
@korthout Oops my bad. Good catch, I'm gonna need a new approval now.